### ISSUE TITLE
Comma in the integration title crashes the integration

### ISSUE THREAD

**Issue by mmuzikar** [syndesisio/syndesis#6227]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description
I created an integration which had a comma in the name, it caused the integration to crash. Without any feedback why it happened or what did I do wrong. Since there's a possibility the user couldn't have access to Openshift and the logs, they wouldn't know what's wrong and most likely wouldn't suspect the integration name to be cause of their troubles.
This is the stacktrace produced:
```
Caused by: javax.management.MalformedObjectNameException: Could not create ObjectName from: org.apache.camel:context=test,,type=context,name="test,". Reason: javax.management.MalformedObjectNameException: Invalid character ',' in key part of property
	at org.apache.camel.management.DefaultManagementNamingStrategy.createObjectName(DefaultManagementNamingStrategy.java:418) ~[camel-core-2.21.0.fuse-740038.jar!/:2.21.0.fuse-740038]
	at org.apache.camel.management.DefaultManagementNamingStrategy.getObjectNameForCamelContext(DefaultManagementNamingStrategy.java:106) ~[camel-core-2.21.0.fuse-740038.jar!/:2.21.0.fuse-740038]
	at org.apache.camel.management.DefaultManagementLifecycleStrategy.onContextStart(DefaultManagementLifecycleStrategy.java:167) ~[camel-core-2.21.0.fuse-740038.jar!/:2.21.0.fuse-740038]
	... 40 common frames omitted
```
I believe there are many other characters that cause this behavior... I've found ':' to be problematic as well with quick search on the interwebz. 
## What I'd expect to happen
Either we shouldn't allow these "problematic" characters in the integration name
![image](https://user-images.githubusercontent.com/46345469/61691164-3e2f5880-ad2b-11e9-9693-507e2f76bb25.png)
Here the Name input should shine red, tell the user what's wrong, disable the save and publish buttons.
OR 
Use commas and other characters just for display, I can imagine someone using : or , in the title.

**Comment by squakez** [comment]
The failure is caused by the [ObjectName](https://docs.oracle.com/javase/8/docs/api/javax/management/ObjectName.html) class which is used for jmx managment. This set of characters is going to provoke failure: `,:*?=`

**Comment by squakez** [comment]
Server side should be fixed by PR #6371 . We may make changes to the allowed integration name pattern. Client side there must be a further check because right now the integration is not created but the GUI is not showing any error neither, hanging on.

![image](https://user-images.githubusercontent.com/23169723/63014720-74c24400-be8f-11e9-9e6a-61fdb4d7ff7b.png)

The error message retrieved is actually the following one:
```
{"errorCode":400,"userMsg":"Please make sure that you're sending the correct parameters","developerMsg":"com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `io.syndesis.common.model.integration.Integration$Builder`, problem: name: Integration name can only contain characters and -_.\n at [Source: (io.undertow.servlet.spec.ServletInputStreamImpl); line: 1, column: 3374]"}
```
